src/pylexibank/commands/check_languages.py

```diff
@@ -27,12 +28,28 @@ def check(ds, args):
         gv = get_glottolog_version(cldf)
         if gv != args.glottolog_version:
             warn('Dataset compiled against Glottolog {0}'.format(gv))
-
+    
     bookkeeping = set(
         l.id for l in args.glottolog.api.languoids() if l.lineage and l.lineage[0][1] == 'book1242')
-    for l in cldf['LanguageTable']:
-        if l['Glottocode'] and l['Glottocode'] in bookkeeping:
-            warn('Language {0} mapped to Bookkeeping languoid {1}'.format(l['ID'], l['Glottocode']))
+    
+    cols_with_values = Counter()  # used for check on empty columns
+    nlanguages = 0
+    for nlanguages, row in enumerate(cldf['LanguageTable'], 1):
```

Comment:
Just nitpicking: I try to always pass optional arguments by name, i.e. `enumerate([], start=1)`. I once got bitten trying to pass flags into `re.split` positionally, forgetting that there's an `maxsplit` argument. That was hard to debug.

Reply:
will do :)